pkger/models.go

```diff
@@ -2656,9 +2656,6 @@ func (c chart) properties() influxdb.ViewProperties {
 				Visible:      fieldOpt.Visible,
 			})
 		}
-		sort.Slice(fieldOptions, func(i, j int) bool {
```

Comment:
the order it comes in needs to respected, the UI is consuming it directly it turns out